boards.txt

```diff
@@ -8397,6 +8270,194 @@ adafruit_metro_esp32s2.menu.EraseFlash.none.upload.erase_cmd=
+adafruit_metro_esp32s3.build.dfu_on_boot=0
+adafruit_metro_esp32s3.build.f_cpu=240000000L
+adafruit_metro_esp32s3.build.flash_size=16MB
+adafruit_metro_esp32s3.build.flash_freq=80m
+adafruit_metro_esp32s3.build.flash_mode=dio
+adafruit_metro_esp32s3.build.boot=qio
+adafruit_metro_esp32s3.build.partitions=default
+adafruit_metro_esp32s3.build.defines=
+adafruit_metro_esp32s3.build.loop_core=
+adafruit_metro_esp32s3.build.event_core=
+adafruit_metro_esp32s3.build.flash_type=qio
+adafruit_metro_esp32s3.build.psram_type=qspi
+adafruit_metro_esp32s3.build.memory_type={build.flash_type}_{build.psram_type}
+
+adafruit_metro_esp32s3.menu.LoopCore.1=Core 1
+adafruit_metro_esp32s3.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+adafruit_metro_esp32s3.menu.LoopCore.0=Core 0
+adafruit_metro_esp32s3.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+adafruit_metro_esp32s3.menu.EventsCore.1=Core 1
+adafruit_metro_esp32s3.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+adafruit_metro_esp32s3.menu.EventsCore.0=Core 0
+adafruit_metro_esp32s3.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+adafruit_metro_esp32s3.menu.USBMode.default=USB-OTG (TinyUSB)
+adafruit_metro_esp32s3.menu.USBMode.default.build.usb_mode=0
+adafruit_metro_esp32s3.menu.USBMode.hwcdc=Hardware CDC and JTAG
+adafruit_metro_esp32s3.menu.USBMode.hwcdc.build.usb_mode=1
+
+adafruit_metro_esp32s3.menu.CDCOnBoot.cdc=Enabled
+adafruit_metro_esp32s3.menu.CDCOnBoot.cdc.build.cdc_on_boot=1
+adafruit_metro_esp32s3.menu.CDCOnBoot.default=Disabled
+adafruit_metro_esp32s3.menu.CDCOnBoot.default.build.cdc_on_boot=0
+
+adafruit_metro_esp32s3.menu.MSCOnBoot.default=Disabled
+adafruit_metro_esp32s3.menu.MSCOnBoot.default.build.msc_on_boot=0
+adafruit_metro_esp32s3.menu.MSCOnBoot.msc=Enabled (Requires USB-OTG Mode)
+adafruit_metro_esp32s3.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+adafruit_metro_esp32s3.menu.DFUOnBoot.default=Disabled
+adafruit_metro_esp32s3.menu.DFUOnBoot.default.build.dfu_on_boot=0
+adafruit_metro_esp32s3.menu.DFUOnBoot.dfu=Enabled (Requires USB-OTG Mode)
+adafruit_metro_esp32s3.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+adafruit_metro_esp32s3.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+adafruit_metro_esp32s3.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+adafruit_metro_esp32s3.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+adafruit_metro_esp32s3.menu.UploadMode.default=UART0 / Hardware CDC
+adafruit_metro_esp32s3.menu.UploadMode.default.upload.use_1200bps_touch=false
+adafruit_metro_esp32s3.menu.UploadMode.default.upload.wait_for_upload_port=false
+
+adafruit_metro_esp32s3.menu.PSRAM.opi=OPI PSRAM
+adafruit_metro_esp32s3.menu.PSRAM.opi.build.defines=-DBOARD_HAS_PSRAM
+adafruit_metro_esp32s3.menu.PSRAM.opi.build.psram_type=opi
+adafruit_metro_esp32s3.menu.PSRAM.enabled=QSPI PSRAM
+adafruit_metro_esp32s3.menu.PSRAM.enabled.build.defines=-DBOARD_HAS_PSRAM
+adafruit_metro_esp32s3.menu.PSRAM.enabled.build.psram_type=qspi
```

Comment:
If my previous comment is true, this part should be removed :)

Reply:
You can test it easily on your new board, but I guess the board will just crash/reboots infinitely.

Reply:
@hathach So its confirmed now, would be better for users to remove this, if the board will sell only with OPI PSRAM.

Reply:
thank you, Will follow your suggestion